jsonschema2md/__init__.py

```diff
@@ -316,9 +325,20 @@ def _construct_description_line(self, obj: dict[str, Any], add_type: bool = Fals
                 description_line.append(_("Cannot contain unevaluated properties."))
 
         if "$ref" in obj:
+            url = urlsplit(obj["$ref"])
```

Comment:
Should we use this? Split the url and parse it's logical parts or should we just `ref.startswith(self.base)`?

Reply:
I like to use strong functions like `urlsplit` :-)